### ISSUE TITLE
Protocollib Error on 1.18.2 Server

### ISSUE THREAD

**Issue by xXGamerHDXTv** [dmulloy2/ProtocolLib#2988]
**Describe the bug**
Console Error and Ingame Kicks for invalid player data... I have a 1.18.2 server and as soon as I download and install the latest version of the plugin and then try to join the server I get kicked because of Invalid Player data, I don't have this error with old versions of the plugin Protocollib. (I don't have this error on Ver: 5.1.0)

Error: https://pastebin.com/uR7icXTQ

Any way I can fix this?

**Comment by dmulloy2** [comment]
does this happen in the 5.2.0 release?

**Comment by xXGamerHDXTv** [comment]
Yes

**Comment by dmulloy2** [comment]
Interesting. Can you run `/protocol dump` and paste the outputs with Pastebin?

**Comment by xXGamerHDXTv** [comment]
https://pastebin.com/f6dpeGJR

**Comment by xXGamerHDXTv** [comment]
Any news regarding this?

**Comment by MasonDye** [comment]
Yes, it's a big bug, my server version is 1.17.1 and I'm sure it's a Protocollib bug.

**Comment by MasonDye** [comment]
This bug also appears in 1.20.1
